src/material/autocomplete/autocomplete.scss

```diff
@@ -16,7 +15,7 @@
   transform-origin: center top;
 
   @include mdc-list.list-base($query: structure);
-  @include cdk.high-contrast(active, off) {
+  @media(forced-colors: active) {
```

Comment:
The existing usages should continue to use the `high-contrast` mixin.

Reply:
I really don't see why it would be better to use cdk.high-contrast mixin when there's a native API that accomplishes the same thing 😕 .

Reply:
Because the mixin allows us to easily change the implementation if a new browser API comes in the future.

Reply:
Okay, so you want use to use high-contrast mixin internally instead of forced-colors native api. Do you think our documentation should say Angular devs should use `cdk.high-contrast` or `@media(forced-colors)`?

Reply:
Yes, I think that the mixin and our guidance should stay the same. Only the internal implementation should change.

Reply:
Okay, I might be missing something here. I thought the point of doing this, is that Angular developers wouldn't have to learn to use one of our APIs, and could use a native API instead. Since we're not creating our own API, we could leverage existing documentation and point to the ecosystem for guidance. But also, existing apps could continue to use `high-contrast` and `HighContrastDetector` for a long time, and they would continue to work correctly. This would improve developer experience, and also not break existing apps. cc @twerske